include/swift/AST/DiagnosticsSema.def

```diff
@@ -2381,6 +2381,9 @@ ERROR(broken_encodable_requirement,none,
       "Encodable protocol is broken: unexpected requirement", ())
 ERROR(broken_decodable_requirement,none,
       "Decodable protocol is broken: unexpected requirement", ())
+// SWIFT_ENABLE_TENSORFLOW
+ERROR(broken_parameter_aggregate_requirement,none,
+      "ParameterAggregate protocol is broken: unexpected requirement", ())
```

Comment:
What do you mean by "broken"? This diagnostic is not understandable by the user.

Reply:
Removed diagnostic and replaced usage with assertions in https://github.com/apple/swift/pull/18140/commits/7768fa9d6d142cf111f4f110d44b68dd1cf52a74.
This is an internal error, I originally copied the diagnostic from other synthesis-related diagnostics.

Reply:
I see! I missed those precedents. Either way is fine I think.